src/pages/patterns/disclosures-pattern/index.mdx

```diff
@@ -0,0 +1,779 @@
+---
+title: Disclosures
+description:
+  Disclosures are small moments that open up on a page and reveal additional
+  information related to the source it is triggered from.
+---
+
+<PageDescription>
+
+Disclosures are small moments that open up on a page and reveal additional
+information related to the source it is triggered from.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Overview</AnchorLink>
+<AnchorLink>Designing with disclosures</AnchorLink>
+<AnchorLink>Profile menu</AnchorLink>
+<AnchorLink>Context menu</AnchorLink>
+<AnchorLink>Settings and filter menus</AnchorLink>
+<AnchorLink>Combo button</AnchorLink>
+<AnchorLink>Related</AnchorLink>
+<AnchorLink>References</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
+
+</AnchorLinks>
+
+## Overview
+
+Disclosures support a wide range of different use cases in product interfaces
+and are commonly used to reveal more information or details about an element or
+content on a page. Disclosures can contain interactive elements while preserving
+accessible functionality.
+
+At its core, a disclosure is comprised of two parts—a trigger that the user
+interacts with by clicking or using their keyboard and the popover that opens
+and discloses the content.
```

Comment:
At a high-level, I think this could be too narrow of a definition for a disclosure. [From the W3C](https://w3c.github.io/aria-practices/#disclosure), we have:

> A disclosure is a widget that enables content to be either collapsed (hidden) or expanded (visible). It has two elements: a disclosure button and a section of content whose visibility is controlled by the button.

And so this shows up in things like accordions, for example, that may not make use of a popover.

We can totally pair up a trigger button and a popover and use the disclosure pattern to toggle its visibility, but didn't want to say that only a popover may be used since the pattern is broader.